### ISSUE TITLE
Please add '**--num_units=128**' to the tutorial's inference command example to be consistent

### ISSUE THREAD

**Issue by chunsheng-chen** [tensorflow/nmt#127]
Please add '**--num_units=128**' to the tutorial's inference command example, 

python -m nmt.nmt \
    --out_dir=/tmp/nmt_model \
    --inference_input_file=/tmp/my_infer_file.vi \
    --inference_output_file=/tmp/nmt_model/output_infer

to be consistent with the tutorial's training command example?

python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    **--num_units=128** \
    --dropout=0.2 \
    --metrics=bleu

**Comment by oahziur** [comment]
@chunsheng-chen I think the problem was introduced by relaxing rules of overriding hparams. This should be fixed after 8854dc87d3f22f0f499b5575204aebd6dd7c04e6 since the saved hparams won't be override by default.
